Sources/App/Core/Search.swift

```diff
@@ -286,8 +319,14 @@ enum Search {
         select = select
             .column(SQLFunction("LEVENSHTEIN", args: repoOwner, SQLBind(mergedTerms)),
                     as: levenshteinDist)
+        select = select
+            .column(ts_rank(vector: tsvector, query: tsquery), as: tsrankvalue)
```

Comment:
I believe this only compiles because `tsvector` and `tsquery` are defined as a constants at `Search` level. We'll need to fill these in.

Reply:
Never mind, I missed the `from` clauses just below.

However, we'll need to keep in mind that multiple `from` clauses will affect the cardinality

Reply:
This is already the case in `keywordMatchQueryBuilder` where we currently unnest the keywords (live version). 

That's potentially an existing bug. Not in the sense that it'll give wrong results I believe but I suspect it selects N(search view rows) x (N keywords) rows without a join and filters on the product when we really want to filter just on keywords.